### PR TITLE
Removed Vertex Class, added commons & improved triangle clas

### PR DESCRIPTION
**Pull request by Carlbaum**

### CODE REVIEW

File: src/geometry/triangle.cc [hunk 1]
```diff
@@ -1,17 +1,15 @@
 #include "triangle.h"
 #include <iostream>
 
-// TODO: Decide which way is best! Here or in .h file????
-// Triangle::Triangle() {}
+Triangle::Triangle(Vertex v1, Vertex v2, Vertex v3) : v1_(v1), v2_(v2), v3_(v3) {
+    color_ = COLOR_WHITE;
```
Comment (novalain) [inline src/geometry/triangle.cc]: Fix indentation to 2 spaces

**Review by novalain — CHANGES_REQUESTED** [review]

**Review by novalain — APPROVED** [review]

File: src/geometry/triangle.cc (hunk 1 above)
Comment (Carlbaum) [reply src/geometry/triangle.cc]: AAAHHHH!

File: src/geometry/triangle.cc [hunk 2]
```diff
@@ -1,17 +1,15 @@
 #include "triangle.h"
 #include <iostream>
 
-// TODO: Decide which way is best! Here or in .h file????
```
Comment (novalain) [inline src/geometry/triangle.cc:4]: Can remove this comment

File: src/geometry/triangle.cc [hunk 3]
```diff
@@ -1,17 +1,15 @@
...
+    CalcNormal();
+}
 
-// Triangle::Triangle(Vertex v1, Vertex v2, Vertex v3) {
-//   v1_ = v1;
-//   v2_ = v2;
-//   v3_ = v3;
-// }
+Triangle::Triangle(Vertex v1, Vertex v2, Vertex v3, glm::vec3 color) : v1_(v1), v2_(v2), v3_(v3), color_(color) {
+    CalcNormal();
```
Comment (novalain) [inline src/geometry/triangle.cc]: Fix indentation

**Review by novalain — CHANGES_REQUESTED** [review]

**Review by novalain — APPROVED** [review]

**Pull request merged**